plottr/apps/inspectr.py

```diff
@@ -129,7 +129,8 @@ def __lt__(self, other: "SortableTreeWidgetItem") -> bool:
 class RunList(QtWidgets.QTreeWidget):
     """Shows the list of runs for a given date selection."""
 
-    cols = ['Run ID', 'Experiment', 'Sample', 'Name', 'Started', 'Completed', 'Records', 'GUID']
+    cols = ['Run ID', 'Tag', 'Experiment', 'Sample', 'Name', 'Started', 'Completed', 'Records', 'GUID']
+    tag_dict = {'': '', 'star': '⭐', 'trash': '🗑️'}
```

Comment:
i wonder whether the "trash bin" icon will give an impression to users that this removes a dataset from the database (which is not possible). So perhaps we can use a different icon, e.g. "thumbs down" or "sad face" or something else?

Reply:
A trash bin 🗑️ was the natural choice for me, a long-time user of LabRAD.
How about a simple cross mark ❌?
An eye with a slash meaning "invisible" might also be good, except there is no unicode character so the code becomes more complicated.

Reply:
yes, the cross seems like a very decent choice :)

Reply:
Great, I've replaced 🗑️ with ❌.